.circleci/config.yml

```diff
@@ -418,14 +418,14 @@ commands:
     steps:
       - restore_cache:
           keys:
-            - rust-target-v2-<< parameters.platform >>-{{ checksum "Cargo.lock" }}
+            - &target-cache-key rust-target-v3-<< parameters.platform >>-{{ .Branch }}-{{ checksum "Cargo.lock" }}
```

Comment:
Btw, it's relatively important that you DO bump to `v3` here because without it, CircleCI does this "tiered disposal" of arguments on the cache key until it gets a hit, which means without the bump, you might still get a previous cache segment based on just _platform_.